boards/nscreen-32.json

```diff
@@ -0,0 +1,45 @@
+{
+  "build": {
+    "arduino":{
+      "ldscript": "esp32_out.ld"
+    },
+    "core": "esp32",
+    "extra_flags": "-DARDUINO_ESP32_DEV",
+    "f_cpu": "240000000L",
+    "f_flash": "40000000L",
+    "flash_mode": "dio",
+    "hwids": [
+      [
+        "0x0403",
+        "0x6010"
+      ]
+    ],
+    "mcu": "esp32",
+    "variant": "esp32"
+  },
+  "connectivity": [
+    "wifi",
+    "bluetooth",
+    "ethernet",
+    "can"
+  ],
+  "frameworks": [
+    "arduino",
+    "espidf"
+  ],
+  "name": "YeaCreate NSCREEN-32",
+  "upload": {
+    "flash_size": "16MB",
+    "maximum_ram_size": 327680,
+    "maximum_size": 16777216,
+    "protocols": [
+      "esptool",
+      "espota",
+      "ftdi"
+    ],

```

Comment:
No need to explicitly specify upload protocols, they're configured in the platform dynamically.

Reply:
ok, do I need to change it , and  pull request again ?

Reply:
Simply push a new commit with a fix

Reply:
ok, please close the PR, I will make a new PR again

Reply:
Put it here:
https://github.com/platformio/platform-espressif32/pull/446